tutor/src/screens/assignment-review/homework-scores.js

```diff
@@ -235,7 +235,7 @@ const TaskResult = observer(({ heading, result, striped }) => {
             isTrouble={result && !result.isUnattemptedAutoZero && result.isTrouble}
             isUnattemptedAutoZero={result && result.isUnattemptedAutoZero}
         >
-            {!heading.isCore && <DroppedQuestionHeadingIndicator heading={heading} />}
```

Comment:
I believe this triangle should only appear if the one on https://github.com/openstax/tutor-js/blob/657effafe37bf87e32857010565f70f5356e1f81/tutor/src/screens/assignment-review/homework-scores.js#L83 does not so we need the !isCore check.